internal/provider/resource_gitlab_project_access_token.go

```diff
@@ -90,17 +90,27 @@ var _ = registerResource("gitlab_project_access_token", func() *schema.Resource
 				Type:        schema.TypeInt,
 				Computed:    true,
 			},
+			"access_level": {
+				Description:      fmt.Sprintf("The access level for the project access token. Valid values are: %s. Default is `%s`.", renderValueListForDocs(validProjectAccessLevelNames), accessLevelValueToName[gitlab.MaintainerPermissions]),
+				Type:             schema.TypeString,
+				ValidateDiagFunc: validation.ToDiagFunc(validation.StringInSlice(validProjectAccessLevelNames, false)),
+				Optional:         true,
+				Default:          accessLevelValueToName[gitlab.MaintainerPermissions],
```

Comment:
@timofurrer @MrLuje This seems to be causing errors in our CI pipeline. Is there a reason to default to maintainer access, or can this default be removed?

```
Error: POST http://127.0.0.1:8080/api/v4/projects/42/access_tokens: 400 {message: 400 Bad request - Could not provision maintainer access to project access token}
```

Reply:
Ah, I saw your comment below

> `// default permission on gitlab side when unspecified`

In this case I think the attribute should be Optional+Computed instead of having a Default

Reply:
Ah, disregard. The CI failure was caused by the expiration date, not the access level.